libr/include/r_util/r_alloc.h

```diff
@@ -52,6 +52,22 @@ R_API void r_free(void *p);
 #define _r_free r_free
 #define _r_realloc r_realloc
 
+static inline void *r_malloc_logerr (size_t n) {
+	void *buf = malloc (n);
+	if (R_UNLIKELY (!buf)) {
+		R_LOG_ERROR ("malloc(%zu) failed", n);
```

Comment:
is `%zu` even portable?

Reply:
%p looks better to me. But maybe worth waiting for 5.7.9 to rework all those